### DISCUSSION TITLE
Is there a way to select only one folder and exclude all others in Smart Connections?

### DISCUSSION THREAD

**Discussion by scottyoung805** [brianpetro/obsidian-smart-connections#354] [Q&A] [answered]
Hi all,

I'm a writer and I've been compiling tons of notes regarding the world-building of my universe in Notion. I'm working to migrate this over into Obsidian and I'd love to be able to ask GPT about items in my notes. I'd also like it to review the manuscript of book one in my series to ask it stupid questions like "What was the name of that planet that character X mentioned in the scene on the space station?". This is to build out my series bible but without the WIP stuff I'm writing inside Obsidian that dives deeper into the worldbuilding until it's more complete.

Basically I'd like to just say, SmartConnections only use this folder to review and chat with me about. Any idea how to do this?

**Comment by brianpetro** [comment, marked as answer]
You can specify a folder inside the chat text box with a `/` (forward-slash). Typing the `/` should also bring up a window for easy folder selection.

**Reply by scottyoung805** [reply]
Thanks. I feel a little dumb for asking the question and greatly appreciate the answer.
